models/product.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose')
+
+const productSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  price: {
+    type: Number,
+    required: true
+  },
+  stock: {
+    type: Number
+  },
+  storeId: {
```

Comment:
I think you should have an actual relation between the storeId and the products collection. check code snippet below @DimejiAre @Rexben001 
` storeId: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Store'
  }`